artifacts/deploy/webhook-configuration.yaml

```diff
@@ -237,3 +237,20 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: [ "v1" ]
     timeoutSeconds: 3
+  - name: resourcedeletionprotection.karmada.io
+    rules:
+      - operations: ["DELETE"]
+        apiGroups: ["*"]
+        apiVersions: ["*"]
+        resources: ["*"]
+        scope: "*"
+    clientConfig:
+      url: https://karmada-webhook.karmada-system.svc:443/validate-resourcedeletionprotection
+      caBundle: {{caBundle}}
+    objectSelector:
+      matchExpressions:
```

Comment:
Just a question:

In which version of Kubernetes was the `objectSelector` field introduced?

Reply:
i found it on kubernetes v1.16 changelog:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md?plain=1#L1624
and this pr shows the objectSlector field
https://github.com/kubernetes/kubernetes/pull/79549/files#diff-b79b31e3158f78ed838a0d030a6801ec70c8acca2dd1ef4081dc8855c0bf81eaR56

Reply:
it will support our compatibility table
https://github.com/kubernetes/kubernetes/pull/79549/files#diff-b79b31e3158f78ed838a0d030a6801ec70c8acca2dd1ef4081dc8855c0bf81eaR56
from v1.16~latest

Reply:
Thanks~